UPGRADE-2.0.md

```diff
@@ -30,3 +30,5 @@ UPGRADE FROM 1.x to 2.0
  * The constructor of `MongoDB\BSON\UTCDateTime` no longer accepts a `string`
    argument. To pass 64-bit integers on 32-bit platforms, use the
    `MongoDB\BSON\Int64` class instead.
+ * The `--with-openssl-dir` configure option has been removed. If using OpenSSL,
+   ensure that it is detected by `pkg-config`.
```

Comment:
I just realized that we never added a warning when using the option in 1.x. Opened [PHPC-2452](https://jira.mongodb.org/browse/PHPC-2452) to track that.